File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars;

    do
    {
        dollars = get_float("How much dollars is owed? $");
    }
    while (dollars < 0.0); //only positive value

    int change = round(dollars * 100);
    int q = 0, d = 0, n = 0, p = 0;

    while (change / 25 > 0) //quarters
    {
        q ++;
        change -= 25;
    }

    while (change / 10 > 0) //dimes
    {
        d ++;
        change -= 10;
    }

    while (change / 5 > 0) //nickels
    {
        n ++;
        change -= 5;
    }

    while (change / 1 > 0) //pennies
    {
        p ++;
        change -= 1;
    }

    printf("%d\n", q + d + n + p);
         
}

File: cash.py
# pegar função get_float e return
from cs50 import get_float

# read the change
change = get_float("Change owed: ")

# checking if the valeu is positive
while change < 0.0:
    change = get_float("Change owed: ")

# rounding change
doll = int(change * 100)

# coins
coins = 0

for i in [25, 10, 5, 1]:
    while int((doll / i)) > 0:
        coins += 1
        doll -= i

# print total
print(coins)